src/main/resources/assets/mmorpg/lang/ru_ru.json

```
@@ -22,44 +33,49 @@
 "_comment": "STATIONS",
 
 	"block.mmorpg.teleporter": "Картографическое устройство",
+	"block.mmorpg.station.cooking": "Станция готовки",
+	"block.mmorpg.station.salving": "Станция утилизации",
+	"block.mmorpg.station.gear_crafting": "Станция снаряжения",
+	"block.mmorpg.station.alchemy": "Алихимическая станция",
+	"block.mmorpg.station.enchanting": "Модификационная станция",
 
 "_comment": "ITEM TOOLTIPS",
 
-	"mmorpg.tooltip.item_modifiable_in_station": "Можно использовать внутри модификационной станции для снаряжения",
-	"mmorpg.tooltip.lootbox": "Что может быть внутри?",
+	"mmorpg.tooltip.item_modifiable_in_station": "Можно использовать внутри модификационной станции",
+	"mmorpg.tooltip.lootbox": "Что внутри?",
 	"mmorpg.tooltip.player_levelup": "Если ваша шкала опыта заполнена, используйте этот предмет для повышения уровня!",
 	"mmorpg.tooltip.capacitor": "Поместите в станцию ремонта, чтобы уменьшить потребление топлива.",
 	"mmorpg.tooltip.capacitor2": "Множитель топлива",
-	"mmorpg.tooltip.socketable_item": "Этот самоцвет можно вставить в гнездо",
+	"mmorpg.tooltip.socketable_item": "Этот самоцвет можно вставить в свободное гнездо",
 
 "_comment": "RARITY ITEM NAMES",
 
 "_comment": "MISC ITEMS",
 
 	"item.mmorpg.jewel.dex": "Изумрудный самоцвет",
 	"item.mmorpg.jewel.str": "Метеоритный самоцвет",
-	"item.mmorpg.jewel.int": "Звездный самоцвет",
-	"item.mmorpg.jewel.unique_crafted": "Уникальный самоцвет рунического мастерства",
+	"item.mmorpg.jewel.int": "Звездная пыль",
```

Долго думал, предлагаю так. И связь с космосом, и по цвету подходит
```suggestion
	"item.mmorpg.jewel.int": "Лунный самоцвет",
```
или поближе к оригиналу, но мне не так нравится
```suggestion
	"item.mmorpg.jewel.int": "Звездный самоцвет",
```

---

Исправлено